src/TorchSharp/NN/Losses.cs

```diff
@@ -12,21 +12,46 @@ namespace TorchSharp.NN
     /// <summary>
     /// Class maintaing the supported loss functions.
     /// </summary>
-    public class Losses
+    public static partial class Functions
     {
 
         [DllImport ("LibTorchSharp")]
-        private static extern IntPtr THSNN_binary_cross_entropy (IntPtr srct, IntPtr trgt, IntPtr wgt, long reduction);
+        private static extern IntPtr THSNN_cross_entropy(IntPtr srct, IntPtr trgt, IntPtr wgt, long ignore_index, bool hasII, long reduction);
 
-        public static Loss binary_cross_entropy (TorchTensor? weigths = null, Reduction reduction = Reduction.Mean)
+        public static Loss cross_entropy_loss(TorchTensor? weigths = null, long? ignore_index = null, Reduction reduction = Reduction.Mean)
         {
             return (TorchTensor src, TorchTensor target) => {
-                var res = THSNN_binary_cross_entropy (src.Handle, target.Handle, weigths?.Handle ?? IntPtr.Zero, (long)reduction);
+                var ii = ignore_index.HasValue ? ignore_index.Value : -100;
```

Comment:
where's the -100 come from?

Reply:
This is the default value in the Torch C++ code for this option. Since we do all the defaults on the C# side of the interface, I thought I should use the same.